### ISSUE TITLE
Contact info block: add labels for accessibility

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#30955]
When editing the form, `textarea` is missing `label`s.

<img width="742" alt="image" src="https://user-images.githubusercontent.com/87168/53243483-a44a7d80-36b0-11e9-9a80-ec8d66af0408.png">

Placeholder isn't sufficient enough for good accessibility.

https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text

**Comment by simison** [comment]
BTW why `textarea rows="1"` and not `input`?

**Comment by enejb** [comment]
I was going for a particular look and use a SimpleInput which produced the output as described above.
In the future I was hoping to use the RichText input which would allow the user to bold or italicize parts of the contact info information.

**Comment by enejb** [comment]
cc @scruffian  and @thomasguillot for suggestions how we can add the labels for the fields. 

I think the end result here would be to make the rendered output look a lot different from the way the user is entering it.

**Comment by enejb** [comment]
@simison do you know if using the aria-label="Address" would be enough here?

**Comment by simison** [comment]
> if using the aria-label="Address" would be enough here?

Yep, that would work too. See it used for both `input` and `RichText` in Gutenberg for example:

https://github.com/WordPress/gutenberg/blob/8181261809f928175514dc00d39c888bc5274d8f/packages/block-library/src/search/edit.js#L21-L40

**Comment by simison** [comment]
Seems like using RichText instead of the custom component would work here without any modifications?
